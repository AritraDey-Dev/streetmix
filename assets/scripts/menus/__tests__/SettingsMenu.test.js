/* eslint-env jest */
import React from 'react'
import { renderWithReduxAndIntl } from '../../../../test/helpers/render'
import { SettingsMenu } from '../SettingsMenu'

jest.mock('../../users/localization', () => {})

describe('SettingsMenu', () => {
  it('renders without crashing', () => {
    const wrapper = renderWithReduxAndIntl(<SettingsMenu />)
    expect(wrapper.asFragment()).toMatchSnapshot()
  })
})
