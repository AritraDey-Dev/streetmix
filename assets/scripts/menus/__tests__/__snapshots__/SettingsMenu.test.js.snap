// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsMenu renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="menu"
  >
    <h2
      class="menu-header"
    >
      Units
    </h2>
    <ul
      class="menu-item-group"
    >
      <li
        class="menu-item "
      >
        Metric units (meters)‎
      </li>
      <li
        class="menu-item "
      >
        Imperial units (feet)‎
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
